python-suds has metadata 2.1 with Obsoletes tag
--- distlib-0.3.0/distlib/metadata.py.orig	2020-03-26 06:51:23.830242814 +0100
+++ distlib-0.3.0/distlib/metadata.py	2020-03-26 20:58:27.061576412 +0100
@@ -95,7 +95,7 @@
 # the metadata. Include them in the tuple literal below to allow them
 # (for now).
 _566_FIELDS = _426_FIELDS + ('Description-Content-Type',
-                             'Requires', 'Provides')
+                             'Requires', 'Provides', 'Obsoletes')
 
 _566_MARKERS = ('Description-Content-Type',)
 
--- distlib-0.3.0/tests/test_metadata.py.orig	2019-04-09 08:00:50.000000000 +0200
+++ distlib-0.3.0/tests/test_metadata.py	2020-03-26 21:32:52.520386861 +0100
@@ -281,8 +281,8 @@
         metadata.set_metadata_version()
         self.assertEqual(metadata['Metadata-Version'], '1.2')
         metadata.set('Obsoletes', 'ok')
-        self.assertRaises(MetadataConflictError,
-                          metadata.set_metadata_version)
+        #self.assertRaises(MetadataConflictError,
+        #                  metadata.set_metadata_version)
 
         del metadata['Obsoletes']
         del metadata['Obsoletes-Dist']
